content/automate/ManageIQ/System/Event/RequestEvent/Request.class/request_completed_with_failure.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: request_completed_with_failure
+    inherits: 
+    description: 
+  fields:
+  - rel5:
+      value: "/System/Policy/request_completed_with_failure"
```

Comment:
@AparnaKarve 
/System/Policy/request_completed_with_failure is not defined in the database in https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/Policy.class are you planning on using the missing instance for this

Reply:
@mkanoor Should I also define request_completed_with_failure under https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/Policy.class ?
The way I have it right now seems to work.